src/filesystem.cpp

```diff
@@ -82,6 +82,19 @@ std::string as_norm_dir( const std::string &path )
 {
     fs::path dir = fs::u8path( path ) / fs::path{};
     fs::path norm = dir.lexically_normal();
+#ifdef _WIN32
```

Comment:
This check is probably not needed since `ensure_valid_file_name` is implemented for and modifies the path on non-Windows systems.